disnake/abc.py

```diff
@@ -1290,9 +1290,16 @@ async def create_invite(
         max_age: :class:`int`
             How long the invite should last in seconds. If it's 0 then the invite
             doesn't expire. Defaults to ``0``.
+
+            .. warning::
+
+                If the guild is not a Community guild (has ``COMMUNITY`` in :attr:`.Guild.features`),
+                this must be set to a time between ``0`` and ``2_592_000`` seconds.
+
         max_uses: :class:`int`
             How many uses the invite could be used for. If it's 0 then there
             are unlimited uses. Defaults to ``0``.
+
```

Comment:
this is not needed

Reply:
removing this turns the html into this 
![image](https://github.com/DisnakeDev/disnake/assets/71233171/a914e996-e8fe-4a72-89c0-72e09fa8e54e)

Reply:
what..... fine

Reply:
oh! I didn't look closely on which line this was commented on!

Reply:
https://github.com/DisnakeDev/disnake/pull/1056/commits/e16ea2a95206e1193563f50cc91fb9feea783da3